network-api/networkapi/wagtailpages/pagemodels/buyersguide/campaign_page.py

```diff
@@ -0,0 +1,131 @@
+        default=False,
+        help_text='For text-heavy pages, turn this on to reduce the overall width of the content on the page.'
+    )
+    body = StreamField(
+        block_types=(('accordion', customblocks.AccordionBlock()),
+                     ('paragraph', RichTextBlock(
+                         features=full_content_rich_text_options,
+                         template='wagtailpages/blocks/rich_text_block.html',
+                     )),
+                     ('card_grid', customblocks.CardGridBlock()),
+                     ('image_grid', customblocks.ImageGridBlock()),
+                     ('iframe', customblocks.iFrameBlock()),
+                     ('image', customblocks.AnnotatedImageBlock()),
+                     ('audio', customblocks.AudioBlock()),
+                     ('image_text', customblocks.ImageTextBlock()),
+                     ('image_text_mini', customblocks.ImageTextMini()),
+                     ('video', customblocks.VideoBlock()),
+                     ('linkbutton', customblocks.LinkButtonBlock()),
+                     ('looping_video', customblocks.LoopingVideoBlock()),
+                     ('pulse_listing', customblocks.PulseProjectList()),
+                     ('single_quote', customblocks.SingleQuoteBlock()),
+                     ('slider', customblocks.FoundationSliderBlock()),
+                     ('spacer', customblocks.BootstrapSpacerBlock()),
+                     ('airtable', customblocks.AirTableBlock()),
+                     ('datawrapper', customblocks.DatawrapperBlock()),
+                     ('typeform', customblocks.TypeformBlock()), )
+    )
+
+    def get_donation_modal_json(self):
+        modals = self.donation_modal_relations.all()
+        modals_json = [m.donation_modal.to_simple_dict() for m in modals]
+        return json.dumps(modals_json)
+
+    content_panels = Page.content_panels + [
+        FieldPanel('header'),
+        SnippetChooserPanel('cta'),
+        InlinePanel('donation_modal_relations', label='Donation Modal', max_num=4),
+        StreamFieldPanel('body'),
+    ]
+
+    settings_panels = Page.settings_panels + [
+        MultiFieldPanel(
+            [
+                FieldPanel('narrowed_page_content'),
+            ],
+            classname="collapsible"
+        ),
+    ]
+
+    translatable_fields = [
+        # Promote tab fields
+        SynchronizedField('slug'),
+        TranslatableField('seo_title'),
+        SynchronizedField('show_in_menus'),
+        TranslatableField('search_description'),
+        SynchronizedField('search_image'),
+        # Content tab fields
+
+        # FIXME: Contingency fix while https://github.com/mozilla/foundation.mozilla.org/pull/7771 is sorted out
+        # TranslatableField('cta'),
+        TranslatableField('title'),
+        TranslatableField('header'),
+        SynchronizedField('narrowed_page_content'),
+        TranslatableField('body'),
+        TranslatableField('donation_modal_relations'),
+    ]
```

Comment:
What do you think about listing the content fields before the settings fields? I feel like that would be more consistent with the Wagtail UI and how you listed the panels above.

Reply:
This makes sense to me as well, updated in latest commit!